Hunted Mobile/Hunted Mobile/Hunted Mobile/Model/InviteKey.cs

```diff
@@ -5,7 +5,8 @@
 namespace Hunted_Mobile.Model {
     public class InviteKey : CustomModelErrorMessages<InviteKey> {
         [Required(ErrorMessage = "De code is verplicht")]
-        [MinLength(3, ErrorMessage = "De code heeft een minimale lengte van 3 karaktes")]
+        [MinLength(4, ErrorMessage = "De code moet bestaan uit 4 karaktes")]
+        [MaxLength(4, ErrorMessage = "De code moet bestaan uit 4 karaktes")]
```

Comment:
Komt dit dan nog overeen met de validatie uit de api? Voorheen stond er ook een minimale lengte van 4 karakters, maar dit had ik juist teruggezet naar 3.

Reply:
Het moet precies 4 zijn (had ik nagevraagd in Discord). 
Ik had het getest en het werkte, dus zou goed moeten zijn.